.github/workflows/build_robotframework_aio.yml

```diff
@@ -288,3 +289,42 @@ jobs:
       run: |
         chmod +x ./tools/git-tag/tag_repos_with_their_version_file.sh
         ./tools/git-tag/tag_repos_with_their_version_file.sh ./config/repositories/publish_pypi_repos.txt
+
+  update-version-aio:
+    name: Update version AIO
+    runs-on: ubuntu-latest
+    needs: [install-test-windows, install-test-linux]
+    if: ${{ ! failure() && ! cancelled() && github.ref_type == 'tag' }}
+
+    steps:
+    - name: Checkout source
+      uses: actions/checkout@v3
+
+    - name: Set Up Git User
+      run: |
+        git config user.email "huavanthong14@gmail.com"
+        git config user.name "Thong Hua"
+
+    - name: Update version AIO on control file
+      run: |
+        echo "AIO_VERSION=${TAG_NAME#[rd]e[vl]/aio/}" >> $GITHUB_ENV
+        . ./include/bash/common.sh && update_debian_control_file
+      
+    - name: Commit Changes
+      run: |
+        tag_branch=htv3hc/feat/tag-aio-$AIO_VERSION-date-$(date +%d.%m.%Y)
```

Comment:
Hi Thong,
this will create a high numbe of branches if I forget to delete them.
I propose to use one branch for automation only. 
This would mean, that you need to rebase the branch in your automation.
automation/task/update_controlfile_version would be a good name.
Thank you,
 Thomas

Reply:
Hi Thomas,

Thank you for your comments.
I have updated it. 
Currently, I only use one branch is automation/task/update_controlfile_version to update control file, and I will checkout that branch on local runner if it not exist, then I will commit it, and push force to override that branch.
```
git checkout -b automation/task/update_controlfile_version 
git commit -m "Auto-branch"
git push -u  -f origin automation/task/update_controlfile_version 
```

With this solution, it will avoid the behavior related to checkout the old branch and rebase on it because there are 2 use case to handle that behavior.
**Case 1:** If the branch is not exist, it use command
```
git checkout -b automation/task/update_controlfile_version 
git commit -m "Auto-branch"
git push -u origin automation/task/update_controlfile_version 
```
**Case 2:** If the branch is exist
```
git checkout automation/task/update_controlfile_version origin/automation/task/update_controlfile_version 
git rebase develop
git commit -m "Auto-branch"
git push -u origin automation/task/update_controlfile_version 
```

If you have any concerns, please tell me.

Thank you